### ISSUE TITLE
Narrator is not announcing names for list items in Add Widgets pane

### ISSUE THREAD

**Issue by krschau** [microsoft/devhome#1323]
### Dev Home version

0.3

### Steps to reproduce the bug

Pre-requisite:
Turn on Narrator using the short cut key "Ctrl + Win + Enter"

Repro steps:
Launch Dev Home application from Windows Search
Navigate to Add Widgets pane and invoke it
Now navigate to list items in the left pane like SSH Keychain, Memory
Observe the behavior of Narrator announcement

http://task.ms/45055407

### Expected result

On navigating to list items, Narrator should announce the corresponding name property associated to it

### Actual result

On navigating to list items in the left navigation pane, Narrator is announcing as "Navigation list item"
